packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -148,3 +158,67 @@ contract OPContractsManager_InternalMethods_Test is Test {
         vm.assertEq(expected, actual);
     }
 }
+
+contract OPContractsManager_Upgrade_Harness is CommonTest {
+    event Upgraded(ISystemConfig indexed systemConfig, address indexed upgrader);
+
+    IProxyAdmin proxyAdmin;
+    address upgrader;
+
+    function setUp() public virtual override {
+        super.disableUpgradedFork();
+        super.setUp();
+        if (!isForkTest()) {
+            // This test is only supported in forked tests, as we are testing the upgrade.
+            vm.skip(true);
+        }
+
+        proxyAdmin = IProxyAdmin(EIP1967Helper.getAdmin(address(systemConfig)));
+        upgrader = proxyAdmin.owner();
+        vm.label(upgrader, "ProxyAdmin Owner");
+    }
+
+    function _getOpChains() internal view returns (OPContractsManager.OpChain[] memory) {
+        OPContractsManager.OpChain[] memory opChains = new OPContractsManager.OpChain[](1);
+        opChains[0] = OPContractsManager.OpChain({ systemConfigProxy: systemConfig, proxyAdmin: proxyAdmin });
+        return opChains;
+    }
```

Comment:
nit, should `opChains` just be a storage variable that is set during `setUp`?

Reply:
yes, fixed.